src/components/Toast/toastStore.ts

```diff
@@ -81,7 +83,7 @@ export const addToast = action(
     const autoClose =
       options.autoClose ??
       (!options.actions && options.type !== ToastType.Loading);
-    const duration = options.duration ?? 4000;
+    const duration = options.duration ?? DefaultToastDuration;
```

Comment:
I kind of like this paradigm: https://github.com/standardnotes/snjs/blob/master/packages/snjs/lib/options.ts#L77

So having defaults defined in one place, then during runtime, expand with defaults: https://github.com/standardnotes/snjs/blob/master/packages/snjs/lib/application.ts#L216